Tests/TestingTests/EntryPointTests.swift

```diff
@@ -30,11 +30,12 @@ struct EntryPointTests {
     }
   }
 
-  @Test("Entry point with WarningIssues feature enabled exits with success if all issues have severity < .error")
-  func warningIssues() async throws {
+  @Test("Entry point without WarningIssues feature enabled exits with success if all issues have severity < .error")
```

Comment:
When I wrote the display name of this test (and the one below), I had considered adding a "feature" concept and that's why I wrote `WarningIssues feature`. But that's not really relevant, and now the tests are both more concerned with the _event stream version_ rather than whether or not a feature is enabled. So maybe we should rephrase the display names? E.g.
```suggestion
  @Test("Entry point using event stream version 0 exits with success if all issues have severity < .error")
```

And below
```swift
@Test("Entry point using event stream version 6.3 propagates warning issues and exits with success if all issues have severity < .error")
```